test/async-object-mapper.test.ts

```diff
@@ -385,6 +385,53 @@ describe(AsyncObjectMapper.name, () => {
     });
   });
 
+  it(`allows picking properties`, async () => {
+    interface Input {
+      firstName: string;
+      middleNames: string[];
+      lastName: string;
+      age: number;
+      taxFileNumber: string;
+    }
+
+    interface Output {
+      firstName: string;
+      lastName: string;
+      age: number;
+      fullName: string;
+    }
+
+    const objectMapper = AsyncObjectMapper.create<Input, Output>()({
+      ...mapFromAsync.pick('firstName', 'lastName', 'age'),
+      fullName: async (input) => `${input.firstName} ${input.lastName}`,
+    });
+
+    expect(await objectMapper.map({
+      firstName: 'Shonky',
+      middleNames: ['The', 'Artful'],
+      lastName: 'Dodger',
+      age: 30,
```

Comment:
![oh-stop-it-you](https://github.com/user-attachments/assets/61f47a06-a9c5-4d10-bec7-0c28ba6142a8)

Reply:
<img src="https://media.tenor.com/CqImYid_v7wAAAAM/lie-lee-mack.gif" width="500" /> 😄